scripts/ec2/networking.sh

```diff
@@ -29,13 +29,13 @@ readonly EC2_FILES='/var/tmp/ec2'
 
 [[ -d $EC2_FILES ]] || mkdir -p $EC2_FILES
 
-# The version 3.1.2 is currently the recommended version.
-SRIOV_DRIVER='ixgbevf-3.1.2.tar.gz'
+# The version 4.2.1 is currently the recommended version.
```

Comment:
Thank you Sir!

I remember having to do this with some versions, which is why we need to confirm, but definitely not a show stopper or otherwise.

I am also not entirely sure why they might have (possibly) retired this module option/parameter. I would need to search through change logs to find out (I will have a look later and report back).